fix: move index to src instead of root

jmcdo29 opened the pull request:
chore: clean up imports and configs

I have a preference of having my imports alphabetized by package name.
You're absolutely welcome to ask me to revert those changes, it's just
a preference of mine. I updated the build command to use the specific
`tsconfig.build.json` that will allow for the base `tsconfig.json` to
reference all ts files, while the build one only references the `src`
directory. Also switched out `rm -rf` for `rimraf` to take into account
those who may be on windows (and as the package was already installed).
All tests still pass, and the Node REPL pulls in the correct functions
when using `require('./')`.

fix #9

jmcdo29 committed:
fix: move index to src instead of root

chore: clean up imports and configs

I have a preference of having my imports alphabetized by package name.
You're absolutely welcome to ask me to revert those changes, it's just
a preference of mine. I updated the build command to use the specific
`tsconfig.build.json` that will allow for the base `tsconfig.json` to
reference all ts files, while the build one only references the `src`
directory. Also switched out `rm -rf` for `rimraf` to take into account
those who may be on windows (and as the package was already installed).
All tests still pass, and the Node REPL pulls in the correct functions
when using `require('./')`.

dependabot[bot] committed:
chore(deps): bump acorn from 5.7.3 to 5.7.4

Bumps [acorn](https://github.com/acornjs/acorn) from 5.7.3 to 5.7.4.
- [Release notes](https://github.com/acornjs/acorn/releases)
- [Commits](https://github.com/acornjs/acorn/compare/5.7.3...5.7.4)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
chore(deps): bump lodash from 4.17.15 to 4.17.19

Bumps [lodash](https://github.com/lodash/lodash) from 4.17.15 to 4.17.19.
- [Release notes](https://github.com/lodash/lodash/releases)
- [Commits](https://github.com/lodash/lodash/compare/4.17.15...4.17.19)

Signed-off-by: dependabot[bot] <support@github.com>

jmcdo29 committed:
Merge pull request #2 from jmcdo29/dependabot/npm_and_yarn/lodash-4.17.19

chore(deps): bump lodash from 4.17.15 to 4.17.19

jmcdo29 committed:
Merge pull request #1 from jmcdo29/dependabot/npm_and_yarn/acorn-5.7.4

chore(deps): bump acorn from 5.7.3 to 5.7.4

yuezk commented:
Hi @jmcdo29 and @johnbiundo, I'm running into the same problem as #9, the `dist` folder wasn't included in the npm package. Can we got this PR merged and publish a new version?

dependabot[bot] committed:
chore(deps): bump node-fetch from 2.6.0 to 2.6.1

Bumps [node-fetch](https://github.com/bitinn/node-fetch) from 2.6.0 to 2.6.1.
- [Release notes](https://github.com/bitinn/node-fetch/releases)
- [Changelog](https://github.com/node-fetch/node-fetch/blob/master/docs/CHANGELOG.md)
- [Commits](https://github.com/bitinn/node-fetch/compare/v2.6.0...v2.6.1)

Signed-off-by: dependabot[bot] <support@github.com>
